cucumber.conf.js

```diff
@@ -11,7 +11,7 @@ const { cleanUpTempFiles } = require('./testHelper/helper.js')
 setDefaultTimeout(60000);
 BeforeAll(async function () {
   global.browser = await chromium.launch({
-    headless: true,
+    headless: true
```

Comment:
Remember to set false before pushing.
```suggestion
    headless: false
```